src/main/interface/msp.c

```diff
@@ -829,6 +829,15 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         }
         break;
 
+    case MSP_PILOT_NAME:
```

Comment:
This should not be added at this stage - as a rule, MSP for new parameters is only added in the next minor release **after** the parameter has been released, to avoid having to do complex differentiations in configurator if the format of the parameter has to be changed. Also, in this case, repurposing `MSP_NAME` to allow for multiple strings to be sent is preferrable to introducing a new MSP command.